### ISSUE TITLE
Adding empty records on a ride roster

### ISSUE THREAD

**Issue by smkeesle** [ctnadovich/randowizard#38]
To recreate:

- Edit the roster for an upcoming ride
- Click the link at the top of the table to add a rider to the event.
- Without selecting a rider, click Save or Save and go back to list.
- Note that a new rider record has been added without a rider. Easy to delete...but still.

**Comment by ctnadovich** [comment]
Yes, that seems fixable, even though the CRUD mechanism for editing
rosters is a stock framework. I think it might be possible to fail the
save if the rider isn't specified.

**Comment by ctnadovich** [comment]
It should be now impossible to insert a record with an empty rider ID. If you do it you get a generic error, which is less than ideal (a weakness of grocery crud 1.x). That's probably good enough.
